rsts/deployment/cluster_config/auth_setup.rst

```diff
@@ -247,6 +271,27 @@ Apply Configuration
 
       kubectl rollout restart deployment/flytepropeller -n flyte
 
+#. Add `/oauth2` path routing to Ingress
+
+   .. prompt:: bash
+
+      kubectl edit ingress -n flyte flyte
+
+   Under `spec.rules.http.paths` add the following if not already present
+
+   .. code-block:: yaml
+
+      - path: /oauth2
+        pathType: ImplementationSpecific
+        backend:
+          serviceName: flyteadmin
+          servicePort: 80
+      - path: /oauth2/*
+        pathType: ImplementationSpecific
+        backend:
+          serviceName: flyteadmin
+          servicePort: 80
+
```

Comment:
I see it in the ingress files [here](https://github.com/flyteorg/flyte/blob/master/kustomize/base/ingress/ingress.yaml#L172-L185) and [here](https://github.com/flyteorg/flyte/blob/master/charts/flyte/templates/common/ingress.yaml#L160-L169)

Reply:
Oh great! I must have been using an older version of the helm chart that did not have it. Will remove